### DISCUSSION TITLE
swift package generate-xcodeproj

### DISCUSSION THREAD

**Discussion by VineFiner** [BinaryBirds/feather#38] [Q&A]
swift package generate-xcodeproj create project can not run

**Comment by tib** [comment]
That's a deprecated command, and you should completely avoid using it, please open the project via the Package.swift file.

**Reply by VineFiner** [reply]
For some reasons, pulling git directly is very slow. So generate the xcode project. Is there any solution?

**Reply by tib** [reply]
Unfortunately no, you have to open the Package.swift file using Xcode.
